mongodb-monitoring/templates/deployment.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: extensions/v1beta1
```

Comment:
`apiVersion: apps/v1beta2` is the latest though not required afaik. `apiVersion: apps/v1` will be in 1.9

Reply:
tried running it on that version but wasn't working 🤔

Reply:
it has a slightly different requirement of the yaml